src/components/Register.js

```diff
@@ -1,48 +1,59 @@
-import { useRef, useState } from "react";
-
+import { useState } from "react";
+import { REGISTER_API } from "./Api";
+import axios from "axios";
+import { Link } from "react-router-dom";
 const Register = () => {
-  // const useRef = useRef();
-  // const errRef = useRef();
 
   const [userDetails, setUserDetails] = useState({
-    fname: "",
-    lname: "",
+    first_name: "",
+    last_name: "",
     email: "",
+    username: "",
     password: "",
-    confirmPassword: "",
+    password2: "",
   });
 
   const handleChange = (e) => {
-    setUserDetails({ ...userDetails, [e.target.name]: e.target.value});
-  }
+    setUserDetails({ ...userDetails, [e.target.name]: e.target.value });
+  };
 
   const handleSubmit = async (e) => {
     e.preventDefault();
+    try {
+      axios.post(REGISTER_API, userDetails);
+    } catch (err) {
```

Comment:
What happens after registering has been successful? Do they just stay on the register page?